sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableServiceClientBuilder.java

```diff
@@ -253,6 +280,7 @@ public TableServiceClientBuilder pipeline(HttpPipeline pipeline) {
      *
      * @return The updated {@link TableServiceClientBuilder}.
      */
+    @Override
```

Comment:
Setting a custom `HttpPipeline` would make it so the builder ignores some values in the `Configuration` object. Should we add a note like you did in other methods on this same class?